lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -134,12 +135,26 @@ PortInfo calyx::getPortInfo(BlockArgument arg) {
 
 /// Returns whether the given operation has a control region.
 static bool hasControlRegion(Operation *op) {
-  return isa<ControlOp, SeqOp, IfOp, WhileOp, ParOp>(op);
+  return isa<ControlOp, SeqOp, IfOp, WhileOp, ParOp, StaticRepeatOp,
+             StaticParOp, StaticSeqOp, StaticIfOp>(op);
+}
+
+/// Returns whether the given operation is a static control operator
+static bool isStaticControl(Operation *op) {
+  if (isa<EnableOp>(op)) {
+    // for enables, we need to check whether its corresponding group is static
+    auto component = op->template getParentOfType<ComponentOp>();
```

Comment:
Is this `template` needed? I believe weird things have happened on windows with this.